README.md

```diff
@@ -208,6 +231,10 @@ end
 # Update the stored sign count with the value from `assertion_response.authenticator_data.sign_count`
 ```
 
+## API
+
+_Pending_
```

Comment:
Will this replace the registration/authentication phases above?

Reply:
No, expected to be a API Reference. Complement to the Usage section.